src/Microsoft.DotNet.Watcher.Tools/Internal/MsBuildFileSetFactory.cs

```diff
@@ -143,15 +141,18 @@ private void EnsureInitialized()
             var projectExtensionsPath = Path.Combine(Path.GetDirectoryName(_projectFile), "obj");
 
             // see https://github.com/Microsoft/msbuild/blob/bf9b21cc7869b96ea2289ff31f6aaa5e1d525a26/src/XMakeTasks/Microsoft.Common.targets#L127
-            var projectExtensionFile = Path.Combine(projectExtensionsPath, Path.GetFileName(_projectFile) + ProjectExtensionFileExtension);
+            var projectExtensionFile = Path.Combine(projectExtensionsPath,
+                Path.GetFileName(_projectFile) + ProjectExtensionFileExtension);
 
             if (!File.Exists(projectExtensionFile))
             {
                 // ensure obj folder is available
                 Directory.CreateDirectory(Path.GetDirectoryName(projectExtensionFile));
 
                 using (var fileStream = new FileStream(projectExtensionFile, FileMode.Create))
-                using (var assemblyStream = GetType().GetTypeInfo().Assembly.GetManifestResourceStream("dotnetwatch.targets"))
+                using (
+                    var assemblyStream =
+                        GetType().GetTypeInfo().Assembly.GetManifestResourceStream("dotnetwatch.targets"))
```

Comment:
nit: actually prefer the previous format. At least move the `var` line inline with `using`.